src/components/colorscale/calc.js

```diff
@@ -20,7 +20,7 @@ module.exports = function calc(trace, vals, containerStr, cLetter) {
 
     if(containerStr) {
         container = Lib.nestedProperty(trace, containerStr).get();
-        inputContainer = Lib.nestedProperty(trace._input, containerStr).get();
+        inputContainer = Lib.nestedProperty(trace, containerStr).get();
```

Comment:
@monfera I really can't see a reason for `calcCalcscale` to be modified in this PR.

Could you elaborate?